guides/micronaut-cloud-database-google/micronaut-cloud-database-google.adoc

```diff
@@ -0,0 +1,149 @@
+external:micronaut-cloud-database-base/start.adoc[]
+* You need a https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account and a GCP project.
```

Comment:
Remove "You need", it's redundant with the header of the list ("you will need the following")